lib/src/components/Sections/index.vue

```diff
@@ -0,0 +1,1922 @@
+<template>
```

Comment:
this component deserves lots of comments to clarify its behaviors and the structural choices made here

Reply:
It already has the same comments that were on the vue-sections library for each element in the template. I am not sure I got what comments are still needed and where.

Reply:
new library is the occasion to revisit and improve.

The overall logic of the component is missing some important gaps in commenting out how it was built etc..

let's not rely on what was done and keep improving